api/websocket/server/server.go

```diff
@@ -570,3 +593,17 @@ func (ws *WsServer) sendInboundRelayMessageToClient(v interface{}) {
 		log.Error("Decode relay message failed")
 	}
 }
+
+// client auth, generate challenge
+func (ws *WsServer) sendClientAuthChallenge(sess *session.Session) {
+	resp := make(map[string]interface{})
+	resp["Action"] = "authChallenge"
+
+	var challenge [64]byte
```

Comment:
Why not a byte slice?

Reply:
i modify it to [32]bytes. 
rand.Reader.read need know how many rand bytes to generate, so we need declare its length explicitly.